mediaQueries/mobile.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./mobileFirstStyles.css">
+</head>
+
+<body>
+    <header>
+        <img src="./rad32.png" class="headerimg" alt="headerimg">
+        <!-- can this be nested under something with block properties already?
+            are imgs better in html or css
+        -->
+        <h1>Tomorrow's Technology, Yesterday's Insights</h1>
+        <h2>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Felis imperdiet proin fermentum leo vel orci porta non. Tempor nec feugiat nisl pretium.
+        </h2>
+        <button id="buttontech">TECHNOLOGY</button>
+    </header>
+
+    <aside class="forDesktop">Top Stories
+        <nav class="justforasideflex">
+            <a class="topStories" href="Stanford">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi tempus iaculis urna id volutpat lacus laoreet non.</a>       
+            <a class="topStories" href="3 Ways">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi tempus iaculis urna id volutpat lacus laoreet non.</a>        
+            <a class="topStories" href="Where to">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</a> 
+            <a class="topStories" href="Chukkas">Chukkas lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</a>  
+        </nav>
+    </aside>
+   
+    <footer class="forMobile" role="contentinfo">
+        <nav>Useful Links
+            <a class="footercontent" href="About">About</a>
+            <a class="footercontent" href="Jobs">Jobs</a>       
+            <a class="footercontent" href="Team">Team</a>       
+            <a class="footercontent" href="Advertise">Advertise</a>
+            <a class="footercontent" href="Terms">Terms</a>     
+<!-- wtf does <a> in <li> do? 
+    instead got rid of ul & li    
+-->
```

Comment:
lol. `ul` and `li` usually go together. you use them when you're making a list of items. most navigation elements will have a list of links. so you can use the `ul` element to signify you're creating an unordered list with `li` list items. Since it's a navigation link, it needs to be clickable and send the user to an internal/external page. That's where the `a` tag comes in. the `a` tag defines a hyperlink. you'll add the link in the `href`.
you'll want it to look like this:
`<nav>
                <ul>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                    <li><a href=""></a></li>
                </ul>
            </nav> `